packages/craco/lib/features/cra-paths/api.js

```diff
@@ -0,0 +1,34 @@
+const { mergeWebpackConfig } = require("./merge-webpack-config");
```

Comment:
This requires a non-existent file (https://github.com/gsoft-inc/craco/tree/039ee001b5b86d3312f86b5b5724c4aad46263b1/packages/craco/lib/features/cra-paths)

Reply:
I assume, it should instead import `../webpack/merge-webpack-config`

Reply:
You're right, but it wasn't used in that file, so I deleted it.